megamek/src/megamek/client/bot/princess/FireControl.java

```diff
@@ -3220,6 +3221,18 @@ Mounted getAntiAirAmmo(final List<Mounted> ammoList,
         return unjamVector;
     }
 
+    /**
+     * Return a "Find Club" action, if the unit in question can find a club.
+     */
+    public FindClubAction getFindClubAction(Entity shooter) {
```

Comment:
Please add Nullable Tag if something can be returned as null for ease of future use.